menuinst/win_elevate.py

```diff
@@ -35,42 +36,119 @@ def isUserAdmin():
         raise RuntimeError("Unsupported operating system for this module: %s" % (os.name,))
 
 
+# Taken from conda/common/_os/windows.py
+if os.name == 'nt':
+    from conda.common.compat import ensure_binary
```

Comment:
I think we need to copy this simple function over too, so we don't depend on `conda`:

```python
def ensure_binary(value):
    try:
        return value.encode('utf-8')
    except AttributeError:  # pragma: no cover
        # AttributeError: '<>' object has no attribute 'encode'
        # In this case assume already binary type and do nothing
        return value
```

Reply:
Good catch. Fixed now.